### PR TITLE
PUB-1062 - AAD on e2e

### PR DESCRIPTION
**Pull request by commitBlob**
### JIRA link ###
https://tools.hmcts.net/jira/browse/PUB-1062

### Change description ###
- use AAD process in e2e
- refactor user id to oid as id doesn't exist on AAD

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by alberto-hortiguela — DISMISSED** [review]

File: charts/pip-frontend/values.stg.template.yaml [hunk 1]
```diff
@@ -3,3 +3,4 @@ nodejs:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
   environment:
+    EXCLUDE_E2E: true
```
Comment (kurt-uk) [inline charts/pip-frontend/values.stg.template.yaml:6]: nooo???

File: src/main/controllers/PendingSubscriptionsController.ts [hunk 2]
```diff
@@ -7,9 +7,10 @@ const subscriptionService = new SubscriptionService();
 
 export default class PendingSubscriptionsController {
   public async get(req: PipRequest, res: Response): Promise<void> {
+    console.log('get', req.user['oid']);
```
Comment (kurt-uk) [inline src/main/controllers/PendingSubscriptionsController.ts:10]: left console log in

**Review by ChrisS1512 — APPROVED** [review]

**Review by kurt-uk — CHANGES_REQUESTED** [review]

File: src/main/controllers/PendingSubscriptionsController.ts (hunk 2 above)
Comment (alberto-hortiguela) [reply src/main/controllers/PendingSubscriptionsController.ts:10]: this could be removed

File: charts/pip-frontend/values.stg.template.yaml (hunk 1 above)
Comment (alberto-hortiguela) [reply charts/pip-frontend/values.stg.template.yaml:6]: we need to exclude by now, tech debt ticket was created

**Review by alberto-hortiguela — APPROVED** [review]

File: charts/pip-frontend/values.stg.template.yaml (hunk 1 above)
Comment (alberto-hortiguela) [reply charts/pip-frontend/values.stg.template.yaml]: my wrong, staging should have it

**Review by kurt-uk — APPROVED** [review]

File: src/main/controllers/PendingSubscriptionsController.ts (hunk 2 above)
Comment (commitBlob) [reply src/main/controllers/PendingSubscriptionsController.ts]: I noticed it as well and removed it along enabling e2e on staging

**Pull request merged**